README.md

```diff
@@ -246,7 +258,9 @@ The relayer consists of the following components:
 - At the global level:
   - P2P app network: issues signature `AppRequests`
   - P-Chain client: gets the validators for a Subnet
-  - JSON database: stores latest processed block for each Application Relayer
+  - Relayer database: stores latest processed block for each Application Relayer
+    - Currently supports Redis and local JSON file storage
```

Comment:
does it by default use JSON file storage

Reply:
Similar to how the other config values specifies the default value, can you add that for storage

Reply:
The default values and override behavior is already documented in the `Configuration` section. This section here is intended to give a high level overview of the architectural components.